.github/workflows/generate_schema_workflow.yml

```diff
@@ -0,0 +1,49 @@
+name: Generate New ANCP BIDS Model for new schema version
+
+on:
+  push:
+    paths:
+      - 'schema/*.json'   # Triggers only when a new schema JSON file is added or modified
```

Comment:
so, an ancpBIDS dev would have to copy a new schema.json from the BIDS spec repo over to ancpBIDS, push to main, then this action/workflow will be triggered?

Reply:
Yes, since i copied the new schema.json file to the schema and ran it, i thought that can be trigger point.